html/manifest/background_color.json

```diff
@@ -0,0 +1,59 @@
+{
+  "html": {
+    "manifest": {
+      "background_color": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest",
```

Comment:
Can you update all mdn_url to the actual URL of the new pages? I guess this one now would be https://developer.mozilla.org/docs/Web/Manifest/background_color